packages/react-utilities/etc/react-utilities.api.md

```diff
@@ -86,6 +86,11 @@ export const divProperties: Record<string, number>;
 // @public
 export const formProperties: Record<string, number>;
 
+// Warning: (ae-forgotten-export) The symbol "SomeEventName" needs to be exported by the entry point index.d.ts
```

Comment:
Is this warning expected?

Reply:
Yes, I intentionally didn't export it since we really really don't want anybody ever using that type directly (it could be exported with an `@internal` annotation and a big warning if the missing type creates problems, but I don't believe it should in our supported typescript versions)